zenodo/modules/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -264,6 +270,7 @@
     "access_right": {
       "description": "Access right for record.",
       "type": "string",
+      "default": "open",
```

Comment:
As above. Specifying ``default`` can be useful for required fields and form generation. The form generation is more useful for deposit, but it shouldn't hurt?